configure.in

```diff
@@ -1145,6 +1142,9 @@ PKG_CHECK_MODULES([FREETYPE2],  [freetype2],
 PKG_CHECK_MODULES([TAGLIB],  [taglib >= 1.8],
   [INCLUDES="$INCLUDES $TAGLIB_CFLAGS"; LIBS="$LIBS $TAGLIB_LIBS"],
   AC_MSG_ERROR($missing_library))
+PKG_CHECK_MODULES([ICU_UC], [icu-uc >= 54.1],
```

Comment:
My thinking was that since we build dependencies ourselves and as you stated some of the distros ship quite old versions. Lets start with a fresh version and build it ourselves that way we also get consistency among the platforms. Main thing we would lose by using older versions is new data such as unicode version, collation, time formats and such.
Going through the release notes there's nothing that I can see that we depend on that is missing at least back to v49. There was some Changes between v48->49 to getNF*Instance() that could cause issues, though I haven't tested this.